README.md

```diff
@@ -181,3 +183,19 @@ If you use this package please acknowledge us by citing our paper.
          eprint = {https://onlinelibrary.wiley.com/doi/pdf/10.1002/mrm.29635},
 }
 ```
+
+## Nvidia GPU tested compatibility
+
+We automatically test KomaMRICore's CPU multi-threading support on the most popular operating systems. Nevertheless, for GPU support the process is more manual (until #147). Here is a summary of our automatic CPU tests and local GPU tests for multiple versions of Julia:
+
+| Julia (OS)                            | CPU                                       | GPU (Nvidia) |
+|:--------------------------------------|:----------------------------------------------:|:------------------:|
+| Julia 1.9 (Windows)  | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
+| Julia 1.9 (Linux)    | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
+| Julia 1.9  (Mac)      | [![][gh-actions-img]][gh-actions-url]          | ➖                |
+| Julia 1.10 (Windows)           | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
+| Julia 1.10 (Linux)             | [![][gh-actions-img]][gh-actions-url]          | ✔️                |
+| Julia 1.10 (Mac)               | [![][gh-actions-img]][gh-actions-url]          | ➖                |
+| Julia 1.11 (Windows)          | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ❌                 |
+| Julia 1.11 (Linux)            | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ❌                 |
+| Julia 1.11 (Mac)              | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ➖                 |
```

Comment:
```suggestion
| Julia 1.11 (Mac)              | [![][gh-actions-nightly-img]][gh-actions-nightly-url]  | ➖                 |
```

if you see any problem with this information, please let us know in the form of a GitHub issue.